dark/meson.build

```diff
@@ -1,7 +1,7 @@
-theme_path = join_paths(get_option('datadir'), 'themes/Greybird-Dark')
-a11y_theme_path = join_paths(get_option('datadir'), 'themes/Greybird-Dark-Accessibility')
-compact_theme_path = join_paths(get_option('datadir'), 'themes/Greybird-Dark-Compact')
-bright_theme_path = join_paths(get_option('datadir'), 'themes/Greybird-Dark-Bright')
+theme_path = join_paths(get_option('datadir'), 'themes/Greybird-dark')
+a11y_theme_path = join_paths(get_option('datadir'), 'themes/Greybird-Dark-accessibility')
```

Comment:
This should probably be "Greybird-dark-accessibility"

Reply:
fixed